pages/common/k6.md

```diff
@@ -0,0 +1,27 @@
+# k6
+
+> A modern load testing tool for developers and testers in the DevOps era.
+
+- Run load test locally:
+
+`k6 run {{script.js}}`
+
+- Run load test locally with set number of virtual users and set duration:
+
+`k6 run --vus {{10}} --duration {{30s}} {{script.js}}`
+
+- Run load test locally with set environment variable:
+
+`k6 run -e {{HOSTNAME=example.com}} {{script.js}}`
+
+- Run load test locally using InfluxDB to store results:
+
+`k6 run --out influxdb={{http://localhost:8086/k6db}} {{script.js}}`
+
+- Login into cloud service using secret token:
```

Comment:
"Login into" stutters. "Login to" is better.

Reply:
changed in 4cbf5349